src/coreclr/build-test.cmd

```diff
@@ -663,51 +663,65 @@ exit /b 0
 REM Compile the managed assemblies in Core_ROOT before running the tests
 :PrecompileAssembly
 
+set AssemblyPath=%1
+set AssemblyName=%2
+
 REM Skip mscorlib since it is already precompiled.
-if /I "%2" == "mscorlib.dll" exit /b 0
-if /I "%2" == "mscorlib.ni.dll" exit /b 0
+if /I "%AssemblyName%" == "mscorlib.dll" exit /b 0
+if /I "%AssemblyName%" == "mscorlib.ni.dll" exit /b 0
 REM don't precompile anything from CoreCLR
-if /I exist %CORE_ROOT_STAGE%\%2 exit /b 0
+if /I exist %CORE_ROOT_STAGE%\%AssemblyName% exit /b 0
```

Comment:
Does this mean that we won't be crossgening System.Private.CoreLib.dll with crossgen2?

Reply:
It probably does and I concur it's something worth discussing moving forward just like the default compilation of S.P.C with legacy Crossgen within build.cmd/sh.

Reply:
I think this is a bit different from using the new crossgen in build.cmd/sh. When we run the build-tests.cmd with the option asking for crossgen2 compilation, is it strange to get the whole framework crossgened with crossgen2 except for the S.P.C. I wonder if we could just add a check here that if we are using crossgen2, we would precompile the S.P.C. too.

Reply:
I agree it's weird. I basically left that in as a pre-existing conditional statement but thinking about this some more, what about throwing this away altogether? It doesn't make too much sense to me in the "new world".

Reply:
That sounds reasonable.